about.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/50feea315e.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <title>About - World Chess Tournament</title>
+</head>
+<body>
+    <header>
+        <nav id="navbar">
+            <div id="hamburger">
+                <img id="hamburger-icon" src="images/hamburger-menu-mobile.svg" alt="hamburger-icon">
+                <img id="cancel-icon" src="images/cancel-icon.png" alt="Cancel icon">
+            </div>
+            
+            <a href="/"><img id="fide-logo" src="images/logo-fide.svg" alt="FIDE logo"></a>
+            <ul class="nav-menu">
+                <li><a href="about.html">About</a></li>
+                <li><a href="#program">Program</a></li>
+                <li><a href="#join">Join</a></li>
+                <li><a href="#sponsors">Sponsors</a></li>
+                <li><a href="#news">News</a></li>
+                <li><a href="#campaign" id="campaign">FWC Campaign</a></li>
+            </ul>
```

Comment:
- [ ] Please consider adding a link to the home page in your about page's navigation bar, so users can navigate back easily to the home page. Currently, there is no button on the About page to navigate to the home page.

This will make your project way more accessible which will improve the user experience 📱  😄 👍